tsconfig.json

```diff
@@ -23,5 +23,9 @@
     "moduleResolution": "node",
     "target": "ES2020",
     "strict": true,
+    "outDir": "build",
```

Comment:
Why specify this here?  Is the parameter supplied by `buildJavaScript` in `build_tasks.js` insufficient?

Reply:
Removed, was testing with `tsc` instead of `npm run build:js`. But not necessary otherwise.